cps/templates/modal_restriction.html

```diff
@@ -3,10 +3,10 @@
   <div class="modal-dialog modal-lg" role="document">
     <div class="modal-content">
       <div class="modal-header">
-        <h4 class="modal-title hidden" id="h1">{{_('Select allowed/denied Tags')}}</h4>
-        <h4 class="modal-title hidden" id="h2">{{_('Select allowed/denied Custom Column values')}}</h4>
-        <h4 class="modal-title hidden" id="h3">{{_('Select allowed/denied Tags of user')}}</h4>
-        <h4 class="modal-title hidden" id="h4">{{_('Select allowed/denied Custom Column values of user')}}</h4>
+        <h4 class="modal-title hidden" id="h1">{{_('Select Allowed/Denied Tags')}}</h4>
+        <h4 class="modal-title hidden" id="h2">{{_('Select Allowed/Denied Custom Columns')}}</h4>
+        <h4 class="modal-title hidden" id="h3">{{_('Select Allowed/Denied Tags')}}</h4>
+        <h4 class="modal-title hidden" id="h4">{{_('Select Allowed/Denied Custom Columns')}}</h4>
       </div>
```

Comment:
I'm not happy with all changes: In general we are talking about visibility restrictions. The 4 lines are selected based on the current location from where the dialog was called.  
Select allowed/denied Custom Column values -> In this dialog you can enter values which have to be matched to a certain content of a custom column and you don't enter names of custom columns, what your suggested text implies. Maybe there is a better word than "value" 
Select allowed/denied Tags of user and Select allowed/denied Custom Column values of user are used if an admin edits the visibility restrictions of a certain user in contrast to the other 2 options, where you edit restrictions for yourself or as template for all users. So from my point of view we can not remove the user part.